qa/run_tests.sh

```diff
@@ -18,6 +22,20 @@ fi
 
 [ -d $log_dir ] || mkdir $log_dir
 
+if [ $SHOW_ENV -eq 1 ]
+then
+  echo '-------------------------------------------------------'
+  echo '----------------- Environment details -----------------'
+  echo '-------------------------------------------------------'
+  echo ''
+  echo '---------------------- nvidia-smi ---------------------'
+  nvidia-smi
+  echo ''
```

Comment:
```suggestion
  echo ''
  echo '---------------------- environment -------------------'
  env
  echo ''
```

Reply:
My only hesitation on this one is that if people submit an issue with output from a run with `SHOW_ENV=1` enabled, they may not want to expose all their environment variables. Maybe we can enable this only for `[ $LOCAL == 0 ] && [ SHOW_ENV ==1 ]`?

Reply:
that sounds like a good idea indeed